AdyenCard/Validators/CardNumberValidator.swift

```diff
@@ -34,7 +40,7 @@ public final class CardNumberValidator: Validator {
     
     /// :nodoc:
     public func maximumLength(for value: String) -> Int {
-        19
+        panLength ?? 19
```

Comment:
Is the 19 the default value for `panLength`? It may be a good idea to set it on the constant declaration.

Reply:
it is max PAN length we know of..

Reply:
What I mean is if no `panLength` is passed as a parameter, then the fallback value is always 19. I only see `panLength` being used within the `maximumLength(for value: String)`

Reply:
If you mean to add it in a constant enum, sure why not :)